index.html

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="Self-Taught Software Developers Website">
   <title>Self-Taught Software Developers</title>
+  <link rel ="icon" type="image/png" href="C:\Users\admin\Pictures\Saved Pictures\favicon-32x32.png">
```

Comment:
The image you are referencing(href) for the favicon is only available locally on your machine.
You should include it in the /img folder (https://github.com/self-taught-software-developers/Self-Taught-Software-Developers.github.io/tree/main/img); Then refer to its folder location within the project.

The image should also follow the naming scheme "stsd-logo.jpg".